eZ/Publish/Core/settings/storage_engines/cache.yml

```
@@ -28,6 +28,99 @@ parameters:
     ezpublish.spi.persistence.cache.inmemory.content.ttl: 300
     ezpublish.spi.persistence.cache.inmemory.content.limit: 100
     ezpublish.spi.persistence.cache.inmemory.content.enable: true
+    ibexa.persistence.cache.tag_prefix: 'ez-'
+    ibexa.persistence.cache.tag_patterns:
```

```suggestion
    ibexa.core.persistence.cache.tag_prefix: 'ez-'
    ibexa.core.persistence.cache.tag_patterns:
```
Q: if you're changing tags, would it make any difference if we changed `ez-` prefix too to e.g. `ibx-`?

---

I will adjust this in the tests and implementation itself.